Assets/AWSIM/Scripts/Sensors/ObjectSensor/Classification.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+
+namespace AWSIM
+{
+public class Classification : MonoBehaviour
```

Comment:
@mackierx111 
Yes it's very generic but it can be used for other purposes too, such as tracking or prediction so I think it is better to be generic.

Reply:
@taikitanaka3 
The class name "Classification" is still too generic, and since the namespace is also AWSIM, users will not understand what "Classification" is.

- Besides PerceptionResultSensor, what other uses do you envision for this component?
- How does Autoware define these in naming? (Is it "Classification"?).

Reply:
> Besides PerceptionResultSensor, what other uses do you envision for this component?
I think some V2I message will use this to tell Ego like a smart pole. (but I don't know it it will be implemented or not)

> How does Autoware define these in naming? (Is it "Classification"?).
https://github.com/tier4/autoware_auto_msgs/blob/tier4/main/autoware_auto_perception_msgs/msg/ObjectClassification.idl
autoware_auto_msgs defines ObjectClassification used in Detected Object message as well as tracked object message and predicted object message.
 https://github.com/tier4/autoware_auto_msgs/blob/tier4/main/autoware_auto_perception_msgs/msg/DetectedObject.idl

@mackierx111 
I think naming can also be ObjectClassification or ObjectClassificationResult too.
What do you think?